storage/innobase/xtrabackup/doc/source/howtos/setting_up_replication.rst

```diff
@@ -193,7 +198,7 @@ At ``TheSlave``, do a full backup:
 
 .. code-block:: console
 
-   TheSlave$ innobackupex --user=yourDBuser --password=MaGiCiGaM \
+   TheSlave$ xtrabackup --user=yourDBuser --password=MaGiCiGaM \
```

Comment:
xtrabackup --user=yourDBuser --password=MaGiCiGaM --backup --slave-info --target-dir=/path/to/backupdir